File: backendbuddyfinv/src/main/java/com/es/backendbuddyfinv/dto/GastoTipoEgresoDTO.java
package com.es.backendbuddyfinv.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class GastoTipoEgresoDTO {
    private String tipoEgreso;   // Ej: "Salarios", "Materiales"
    private Double montoTotal;  // SUM(e.costo)
}

File: frontendbuddyfinv/src/models/GastoTipoEgresoDTO.js
// models/GastoTipoEgresoDTO.js
export class GastoTipoEgresoDTO {
    constructor({ tipoEgreso, montoTotal }) {
      this.tipoEgreso = tipoEgreso;   // Ej: "Salarios", "Materiales"
      this.montoTotal = montoTotal;   // SUM(e.costo)
    }
  }